app/build.gradle

```
@@ -20,7 +20,7 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled true
+            minifyEnabled false
```

there is no need to make it false 
if the project gets bigger it will shrink our build size 
```suggestion
            minifyEnabled true
```